pyproject.toml

```diff
@@ -39,7 +39,8 @@ classifiers = [
 requires-python = '>=3.7'
 dependencies = [
     'pydantic@git+https://github.com/pydantic/pydantic.git@main',
-    'phonenumbers'
+    'phonenumbers>=8.13.11,<9',
+    'pycountry@git+https://github.com/yezz123/pycountry.git@main',
```

Comment:
Here i fork the project to update it (latest changes)

Reply:
I don't understand this part. Are you going to release the package on a different namespace? Why not the original package?